pulumi/grapl/Pulumi.local-grapl.yaml

```diff
@@ -25,3 +25,13 @@ config:
   kafka:bootstrapServers:
     - "kafka-broker:9092"
   kafka:tlsEnabled: false
+
+  grapl:artifacts:
+    # You probably don't want any service versions in here.
+
+    # You DO need to specify AMIs (they're require_artifact()'d)
+    # Since this is local-grapl, we don't need real AMIs for Localstack EC2
+    grapl-nomad-consul-client:
```

Comment:
can these just be `nomad-consul-client` and `nomad-consul-server` since they're already namespaced to `grapl` on L29?

Reply:
These two strings are used all over the pipeline - always as an enum of <one of these two names> - and there are places where this is not namespaced (most notably, the `ami_name=` in `image.pkr.hcl`.)